proposals/NNNN-GetSystemTime.md

```diff
@@ -0,0 +1,63 @@
+# GetSystemTime RPC
+
+* Proposal: [SDL-NNNN](NNNN-GetSystemTime.md)
+* Author: [Aleksandr Stasiuk](https://github.com/AStasiuk)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / RPC]
+
+## Introduction
+
+This proposal is about adding a new RPC GetSystemTime to SDL Core for get real system time during security TLS handshake.
+
+
+## Motivation
+
+In some cases SDL Core might not have access to the real system time so needed an interface to the system in order to request it when it is required. System time is necessary for security certificates validation and successful finishing security TLS handshake in case using of Mobile Navigation application on mobile phone.
+
+
+## Proposed solution
+
+
+The solution proposes adding new RPC between SDL and HMI.  GetSystemTime gives possibility to get the real system time status by introducing new API.

```

Comment:
Suggested Text:
The solution proposes adding a new RPC between SDL and HMI called `GetSystemTime`. This RPC allows SDL Core to query the HMI for the accurate system time. 
 
 SDL Core must send the `GetSystemTime` RPC to HMI for obtaining the system time during  Security TLS Handshake.
 
 The expectation is that HMI will provide the accurate system time with the `GetSystemTime` response.